3.1.2

```diff
@@ -0,0 +1,8 @@
++ mocha@8.3.2
+added 77 packages from 49 contributors and audited 281 packages in 5.306s
+
+18 packages are looking for funding
+  run `npm fund` for details
+
+found 6 low severity vulnerabilities
+  run `npm audit fix` to fix them, or `npm audit` for details
```

Comment:
I've removed these files now:

- 3.1.2
- npm
- .idea/
- cypress/integration/login.spec.js - this was a file that was initially checkedin by Evangeline and since I'm not using it, I've deleted it
- node_modules/